types/auth0/index.d.ts

```diff
@@ -214,6 +214,8 @@ export interface PermissionPage extends Page {
 
 export type Grant = 'authorization_code' | 'client_credentials' | 'implicit' | 'password' | 'refresh_token';
 
+export type ActionTrigger = 'post_login' | 'credentials_exchange' | 'pre_user_registration' | 'pre_user_registration' | 'post_user_registration' | 'post_change_password' | 'send_phone_message';
```

Comment:
it's just a type symbol export, is that consumed somewhere within existing declaration?